beetmoverscript/docker.d/init_worker.sh

```diff
@@ -59,24 +58,41 @@ case $COT_PRODUCT in
         exit 1
         ;;
     esac
-    export TASKCLUSTER_SCOPE_PREFIX="project:comm:thunderbird:releng:${PROJECT_NAME}:"
     ;;
   mobile)
-    test_var_set 'MAVEN_ID'
-    test_var_set 'MAVEN_KEY'
-    test_var_set 'MAVEN_NIGHTLY_ID'
-    test_var_set 'MAVEN_NIGHTLY_KEY'
-    export TASKCLUSTER_SCOPE_PREFIX="project:mobile:android-components:releng:${PROJECT_NAME}:"
+    case $ENV in
+      dev|fake-prod)
+        test_var_set 'MAVEN_ID'
+        test_var_set 'MAVEN_KEY'
+        test_var_set 'MAVEN_NIGHTLY_ID'
+        test_var_set 'MAVEN_NIGHTLY_KEY'
+        test_var_set 'DEP_ID'
```

Comment:
Note to self: I need to add these in SOPS  + redeploy dev workers

Reply:
We also need to update the tests to populate fake values here.

Reply:
> Note to self: I need to add these in SOPS + redeploy dev workers

I've populated with fake values the `DEP_ID` and `DEP_KEY` values and redeployed (I foced redeploy to skip the tox blocking dep) `dev-mobile-beetmoverworkers` so that they contain the HEAD of this PR. Locally this works against https://gist.github.com/MihaiTabara/594bf50c4000b7909f01db7fa7f1760b, with the caveat of 401 against S3, of course. I have the production credentials from Jeremy, but he's going to follow-up with the staging ones in the next days too. 

I'll address the rest of the leftovers TODOs from this PR when I'm back from PTO, but for now, this should be enough to unblock @ahal's testing to see a beetmover red task. CC @escapewindow

Reply:
I believe @hneiva has already overwritten the dep keys in SOPS so we should be good on this section.